install/build_gsky.sh

```diff
@@ -0,0 +1,55 @@
+set -xeu
```

Comment:
We do not need this script as building GSKY binary will be handled by the Makefile.